packages/@aws-cdk/aws-appmesh/lib/route-spec.ts

```diff
@@ -107,6 +178,51 @@ export interface GrpcRouteSpecOptions {
    * List of targets that traffic is routed to when a request matches the route
    */
   readonly weightedTargets: WeightedTarget[];
+
+  /**
+   * The retry policy
+   * @default - no retry policy
+   */
+  readonly retryPolicy?: GrpcRetryPolicy;
+}
+
+/** gRPC retry policy */
+export interface GrpcRetryPolicy extends HttpRetryPolicy {
+  /**
+   * gRPC events on which to retry
+   * @default - no retries for gRPC events
+   */
+  readonly grpcRetryEvents?: GrpcRetryEvent[];
+}
+
+/**
+ * gRPC events
+ */
+export enum GrpcRetryEvent {
+  /**
+   * Request was cancelled
+   */
+  CANCELLED = 'cancelled',
+
+  /**
+   * The deadline was exceeded
+   */
+  DEADLINE_EXCEEDED = 'deadline-exceeded',
+
+  /**
+   * Internal error
+   */
+  INTERNAL = 'internal',
+
+  /**
+   * Resource was exhausted
+   */
+  RESOURCE_EXHAUSTED = 'resource-exhausted',
+
+  /**
+   * Unavailable
```

Comment:
What is "unavailable"? The resource, the server we're calling, both, either? Seems like a bigger explanation is needed here.

Reply:
I believe that it means, "The service is unavailable." I think it's pretty generic, in that it could be both or either, but I'm not sure here. The GrpcRetryPolicy docs [don't reveal much](https://docs.aws.amazon.com/app-mesh/latest/APIReference/API_GrpcRetryPolicy.html) and the [CFN docs](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-appmesh-route-grpcretrypolicy.html#cfn-appmesh-route-grpcretrypolicy-grpcretryevents) don't even mention the event.

Reply:
Let's put "service unavailable" then, and a link to that gRPC docs.